### PR TITLE
[9.0] Remove system instances 

### PR DESCRIPTION
**Pull request by atsareg**
This PR removes the concept of system instances. The corresponding configuration options and sections are removed. 

  Requires #7186

BEGINRELEASENOTES

NEW: Remove the use of Setup and System Instances from the configuration and code

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py [hunk 1]
```diff
@@ -1100,44 +1094,6 @@ def do_revert(self, args):
         else:
             gLogger.notice("Software reverted to", result["Value"])
 
-    def do_add(self, args):
```
Comment (fstagni) [inline src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py:1103]: Why did you remove this entirely?

**Review by fstagni — COMMENTED** [review]
Congrats!

File: src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py (hunk 1 above)
Comment (atsareg) [reply src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py:1103]: This call is to add a system instance. The system instances elimination is the whole purpose of this PR. This call is no more relevant

**Comment by atsareg** [comment]
The failed integration tests does not seem to be failing due to this PR changes. @fstagni : can you please have a look, may be you will have an idea for the failure reason ?

**Comment by fstagni** [comment]
> The failed integration tests does not seem to be failing due to this PR changes. @fstagni : can you please have a look, may be you will have an idea for the failure reason ?

Probably not. https://github.com/DIRACGrid/DIRAC/pull/7843 needs to be merged first.

**Review by fstagni — APPROVED** [review]
